plugins/attachment-resources/src/components/AttachmentStyleBoxEditor.svelte

```diff
@@ -102,6 +105,17 @@
     saveTrigger = setTimeout(() => save(saveObject, saveDescription), 2500)
   }
 
+  onDestroy(async () => {
+    // If change text and use mention link inside TextArea
```

Comment:
Nope, you need in case of haveUnsavedChanges call save(saveObject, saveDescription)

Reply:
Nope, we are not in THAT case. The reason this case was that save is async at the start descriptionBox is valid reference, but while await updateAttribute it become invalid one. save already when needed. 

But, I still came up with cleaner solution, because the order is not relevant we could just call createAttachments while reference is still valid.